pkg/neonvm/controllers/vm_cert_controller.go

```diff
@@ -0,0 +1,341 @@
+package controllers
```

Comment:
suggestion: `vm_controller.go` and `vmmigration_controller.go` are named as such because they define the `Controller` for these types. Given it's not quite the same for certificates, maybe let's drop `controller` from the filename?